chromaui/addon-visual-tests
# UI: Render the panel in JSX (but hidden) when inactive

# What I did

- I added a prop `hidden` to the container component used for the Panel.
- A wrapped the content in another one of those wrappers, because that seemed like the easiest way to get this done quick. I didn't want to make changes all over the place at this time.

I could remove the double container, but I'd have to touch something like 10 components and possibly a similar number of story files (add decorators if needed).
I opted not to do that right now.

# How to test

- open storybook in the repo
- open the actions panel
- refresh the page
- the visual tests addon panel should not be shown
- assert that state does get set (easy way to verify is to look at the network tab)
